### ISSUE TITLE
HW2 dataset 관련 문의

### ISSUE THREAD

**Issue by jhshim0326** [swsnu/bd2018#20]
이번 수업으로 deep-learning을 처음 접하다보니

분산시스템에 알맞는 데이터셋을 고르는데 좀 어려움이 있네요.

혹시 데이터셋을 좀 추천해주실 수 있을까요?

**Comment by gyeongin** [comment]
분산 환경에 알맞는 데이터셋이면 더 좋겠지만, 작은 데이터셋을 사용하셔도 좋습니다.
적당한 데이터셋 / 모델을 두 세가지 정도 찾아 추가로 comment 남기도록 하겠습니다.

**Comment by gyeongin** [comment]
과제의 조건이 **No image classification / object recognition / object detection models**이다 보니, deep learning을 처음 배우면 가장 처음 다루는 모델들인 MNIST를 이용한 MLP, LeNet, CIFAR-10이나 ImageNet을 이용한 AlexNet 등은 안타깝게도 사용하실 수 없습니다. 😥

CNN / LSTM에 대한 간단한 개념, Training / Validation / Test dataset에 대한 개념, loss function과 gradient descent에 대한 개념 정도는 작성된 TF 학습 코드를 이해하는데 도움이 많이 된다고 생각하여,
몇 가지 도움이 될 링크를 첨부합니다:
http://cs231n.github.io/classification/
http://cs231n.github.io/optimization-1/
http://cs231n.github.io/convolutional-networks/
http://colah.github.io/posts/2015-08-Understanding-LSTMs/

Deep learning을 처음 접하시는 입장이시니 비교적 간단한 모델들을 대상으로 하는 것이 좋을 것 같습니다.
* https://github.com/tensorflow/models/tree/master/tutorials
위 링크는 TensorFlow tutorial (https://www.tensorflow.org/tutorials/) 에서 사용하고 있는 모델들을 모아 둔 것이기에 tutorial을 따라가며 모델 코드를 이해하고, 이를 분산 수행하는 것에 대한 어려움이 적을 것 같습니다.

이외에도 참고하시면 좋을 모델 링크들은 다음과 같습니다:
* https://github.com/tensorflow/models/tree/master/research
* https://github.com/tensorflow/tensor2tensor/ (https://ai.googleblog.com/2017/06/accelerating-deep-learning-research.html)
* https://github.com/floodsung/Deep-Learning-Papers-Reading-Roadmap
